src/constants/common.ts

```diff
@@ -1,30 +1,3 @@
-import { EndowDesignation } from "types/aws";
-import { IconType } from "components/Icon";
-import { ImageProps } from "components/Image";
```

Comment:
as much as possible, `/constants` should be top-level, just imports from `./types` or third party libraries. This would be helpful if we somehow decide to package `/constants` folder

Reply:
Agreed, unless there's a good reason to split up a logical grouping of a larger number of constants. :+1: